lib/twilio/call.ts

```diff
@@ -922,17 +922,12 @@ class Call extends EventEmitter {
       call_sid: this.parameters.CallSid,
       dscp: !!this._options.dscp,
       sdk_version: C.RELEASE_VERSION,
-      selected_region: this._options.selectedRegion,
```

Comment:
Should we publish the selected edge instead?

Reply:
We already do it here https://github.com/twilio/twilio-voice.js/blob/master/lib/twilio/device.ts#L999